api/leetcode.js

```diff
@@ -94,6 +97,30 @@ export class Leetcode {
         };
     };
 
+    /*
+        Get random problem from category message embed
+    */
+    getRandomProblemFromCategoryMessage = async (category) => {
+        const randomProblemObj = await getRandomProblemFromCategory(category);
+
+        if (!randomProblemObj) {
+            logger.error("Unable to fetch problemSet");
+            return;
+        }
+
+        const categories = await getProblemCategories();
+        const categoryName = Object.keys(categories).find((key) => categories[key] === category);
```

Comment:
interesting, maybe I'm missing something here, why is it not possible to do `categories[key]` ?

Reply:
It's because the key is actually the value..I'll swap in the data structure and do the lookup as you mention.